modules/update-upgrading-cli.adoc

```diff
@@ -123,11 +119,10 @@ $ oc get clusterversion
 .Example output
 [source,terminal]
 ----
-
 Cluster version is <version>
 
 Upstream is unset, so the cluster will use an appropriate default.
-Channel: stable-4.10 (available channels: candidate-4.10, candidate-4.11, eus-4.10, fast-4.10, fast-4.11, stable-4.10)
+Channel: stable-version (available channels: candidate-version, eus-version, fast-version, stable-version)
```

Comment:
Down below, you're using `<version>`.  I'm agnostic about whether you want to mark these placeholders or not, but whichever way you go, it's probably worth being consistent.